From d48f2891bd7fb883a070f520494327d3e46b3ded Mon Sep 17 00:00:00 2001
From: AsavarTzeth <asavartzeth@gmail.com>
Date: Mon, 1 Jul 2019 17:27:58 +0200
Subject: [PATCH 5/5] Add OARS metadata

This is now enforced by Flathub.

By using OARS 1.0 all distributions should be supported. Version 1.1
should work almost everywhere, but there are a few notable distributions
that lack GNOME Software >= 3.27.3 , such as the current OpenSUSE Leap.

Supposedly it makes no difference in this case since for the type:
"Application that can connect to the Internet" the OARS generator asks
the same exact questions in both cases.

The only question here is if downloading favicons could be considered
mild ("Mild: Using any online API, e.g. a user-counter") information
sharing?

Supposedly Google or DuckDuckGo could at the very least store your IP
address, and correlate the icon you downloaded with whatever other data
they have on that IP.
---
 share/linux/org.keepassxc.KeePassXC.appdata.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/share/linux/org.keepassxc.KeePassXC.appdata.xml b/share/linux/org.keepassxc.KeePassXC.appdata.xml
index b8e8c36d..76e59944 100644
--- a/share/linux/org.keepassxc.KeePassXC.appdata.xml
+++ b/share/linux/org.keepassxc.KeePassXC.appdata.xml
@@ -447,4 +447,5 @@
             </description>
         </release>
     </releases>
+    <content_rating type="oars-1.0" />
 </component>
-- 
2.22.0

